core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -242,13 +319,13 @@ object ConsumerGroupCommand {
       groups.asScala.foreach { group =>
         try {
           if (AdminUtils.deleteConsumerGroupInZK(zkUtils, group))
-            println("Deleted all consumer group information for group %s in zookeeper.".format(group))
+            printMessage("Deleted all consumer group information for group '$group' in zookeeper.")
```

Comment:
Looks like we're missing the 's' at the start of the string. Another one below in the `catch`.